lib/aruba/api.rb

```diff
@@ -815,7 +814,7 @@ def assert_no_partial_output(unexpected, actual)
     # @return [TrueClass, FalseClass]
     #   If output of interactive command includes arg1 return true, otherwise false
     def assert_partial_output_interactive(expected)
-      unescape(_read_interactive).include?(unescape(expected)) ? true : false
+      unescape(last_command.stdout).include?(unescape(expected)) ? true : false
```

Comment:
Running a interactive command is not special anymore. Instead those methods use the last command executed. I think this makes the code more readable and easier to maintain.